$OpenBSD: patch-src_Main_cxx,v 1.1 2015/10/28 19:01:39 dcoppa Exp $

Unbreak building mpd with option '--disable-inotify' after commit
f066bb7716200a83e209d27f6b4f87f012033266 ("remove unused Domain
variables")

--- src/Main.cxx.orig	Fri Oct 16 18:07:10 2015
+++ src/Main.cxx	Wed Oct 28 19:47:53 2015
@@ -629,9 +629,6 @@ static int mpd_main_after_fork(struct options options)
 					 *instance->update,
 					 config_get_unsigned(CONF_AUTO_UPDATE_DEPTH,
 							     INT_MAX));
-#else
-		FormatWarning(main_domain,
-			      "inotify: auto_update was disabled. enable during compilation phase");
 #endif
 	}
 #endif
